src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -71,8 +73,10 @@
           style={[
             styles.navContainer(isHeaderHide),
             {
-              height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER,
-              backgroundColor: COLORS.almostBlack
+              height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER + additionalHeaderHeight,
+              backgroundColor: COLORS.almostBlack,
+              borderBottomWidth: isHeaderHide ? 1 : 0,
+              borderBottomColor: COLORS.gray310
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here are aimed at improving the visual appearance of the navigation bar. The height of the navigation bar is now dynamically adjusted based on the `additionalHeaderHeight`. Also, the border bottom width and color are conditionally set based on `isHeaderHide`. These changes enhance the UI's adaptability to different states.
```diff
+               height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER + additionalHeaderHeight,
+               backgroundColor: COLORS.almostBlack,
+               borderBottomWidth: isHeaderHide ? 1 : 0,
+               borderBottomColor: COLORS.gray310
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->